frontend/src/app/commonComponents/Modal.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import ReactModal from "react-modal";
+
+import iconClose from "../../../node_modules/uswds/dist/img/usa-icons/close.svg";
+
+const Header: React.FC<{}> = ({ children }) => (
+  <h3 className="modal__heading">{children}</h3>
+);
+const Footer: React.FC<{}> = ({ children }) => (
+  <h3 className="modal__footer">{children}</h3>
+);
+
+interface Props {
+  onClose: () => void;
+  showModal: boolean;
+}
+interface SubComponents {
+  Header: typeof Header;
+  Footer: typeof Footer;
+}
+
+const Modal: React.FC<Props> & SubComponents = ({
```

Comment:
Ah, a sorely-needed `Modal` component! Would be great to have a tech debt ticket in the near future to align all modals in the app using this component.